src/ui/components/address-action/AddressActionDetails/AddressActionDetails.tsx

```diff
@@ -76,3 +77,46 @@ export function AddressActionDetails({
     </>
   );
 }
+
+/**
+ * TODO: Temporary helper, later the whole AddressActionDetails
+ * must be refactored to take as few params as possible
+ * and to derivce as much data as possible from `addressAction`
+ */
+export function AddressActionComponent({
+  address,
+  addressAction,
+  showApplicationLine,
+  vGap = 16,
+}: {
+  address: string;
+  addressAction: AnyAddressAction;
+  showApplicationLine: boolean;
+  vGap?: number;
+}) {
+  const recipientAddress = addressAction.label?.display_value.wallet_address;
+  const actionTransfers = addressAction.content?.transfers;
+  const singleAsset = addressAction.content?.single_asset;
+  const { networks } = useNetworks();
+
+  if (!networks) {
+    return null;
+  }
+
+  return (
+    <VStack gap={vGap}>
```

Comment:
AddressActionComponent is already inside VStack in SendTransaction.tsx.
Should we remove it there?

Reply:
No, i don't like this pattern when it's wrapped outside, it's too implicit